src/frontend/src/components/public/PIASubHeader/index.tsx

```diff
@@ -120,6 +121,37 @@ function PIASubHeader({
     pia?.review?.programArea?.selectedRoles?.length,
     pia?.status,
   ]);
+
+  useEffect(() => {
+    let reviewProgramAreaDone = false;
+    if (pia?.status === PiaStatuses.FINAL_REVIEW) {
+      const selectedRoles = pia?.review?.programArea?.selectedRoles || [];
+      for (const role of selectedRoles) {
+        if (
+          pia?.review?.programArea?.reviews?.[role]?.isAcknowledged !== true
+        ) {
+          reviewProgramAreaDone = false;
+          break;
+        }
+        reviewProgramAreaDone = true;
+      }
+    }
+
+    if (
```

Comment:
please add this in a separate function with proper commenting.
If  all reviews are acknnowledged
the currest status is in final review and
MPO too has acknowledged the PIA can progress to complete.

Reply:
the useEffect is kind of a function, but I will add more comments